CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing
```

Comment:
This page should actually look like this now:
https://github.com/screwdriver-cd/generator-screwdriver/blob/master/CONTRIBUTING.md